keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -172,10 +172,10 @@ Kubelet components.
 
 If the hints are not compatible, the Topology Manager may choose to
 reject the pod. Behavior in this case depends on a new Kubelet configuration
-value to choose the topology policy. The Topology Manager supports two
-modes: `strict` and `preferred` (default). In `strict` mode, the pod is
-rejected if alignment cannot be satisfied. The Topology Manager could
-use `softAdmitHandler` to keep the pod in `Pending` state.
+value to choose the topology policy. The Topology Manager supports three
+modes: `none`(default), `best-effort` and `restricted`. In `restricted` mode, the pod is
+rejected if alignment cannot be satisfied, alignment in this case means all resources being available on a single NUMA Node. 
```

Comment:
In modern CPUs, non-memory resource might be equally aligned to several NUMA nodes at the time.

Reply:
There is a discussion ongoing here on this: https://github.com/kubernetes/kubernetes/pull/80570#discussion_r317318672